Rubberduck.Core/Common/ClipboardWriter.cs

```diff
@@ -59,5 +75,48 @@ public void Flush()
                 _data = null;
             }
         }
+
+        public void AppendInfo<T>(ColumnInfo[] columnInfos,
+            IEnumerable<T> results,
+            string title,
+            ClipboardWriterAppendingInformationFormat appendingInformationFormat) where T : IExportable
+        {
+            object[][] resultsAsArray = results.Select(result => result.ToArray()).ToArray();
+
+            var includeXmlSpreadsheetFormat = (appendingInformationFormat & ClipboardWriterAppendingInformationFormat.XmlSpreadsheetFormat) == ClipboardWriterAppendingInformationFormat.XmlSpreadsheetFormat;
+            if (includeXmlSpreadsheetFormat)
+            {
```

Comment:
Can't we use `HasFlag` here?

Reply:
Yes. Did not know that was available. TYVM for exposing me to that functionality.